Strata/DDM/Util/Graph/TopSort.lean

```diff
@@ -1,107 +0,0 @@
-/-
-  Copyright Strata Contributors
-
-  SPDX-License-Identifier: Apache-2.0 OR MIT
--/
-
--- Topological sort implementation (not currently used)
-import Strata.DDM.Util.Fin
-import Strata.DDM.Util.Vector
-
-namespace Strata
-
-structure Graph where
```

Comment:
I take note that this file was deleted. I'm surprised it is not used. Perhaps it was meant to be used and did not make it? Anyway.

Reply:
I think this was one of the first things I wrote for code generation, but the module for Tarjan's algorithm replaced it and I kept it around longer than needed.